mlflow/models/utils.py

```diff
@@ -264,3 +263,17 @@ def plot_lines(data_series, xlabel, ylabel, legend_loc=None, line_kwargs=None):
     ax.set(xlabel=xlabel, ylabel=ylabel)
 
     return fig, ax
+
+
+def create_wheeled_model(artifact_path, model_uri):
```

Comment:
Nit: I think we should swap these arguments, since `model_uri` is an input location and `artifact_path` is an output location